### ISSUE TITLE
nuxt build error in 1.5.0

### ISSUE THREAD

**Issue by masayuki0220** [nuxt-community/nuxt-logrocket#152]
Hello everyone.

There is a build error in version 1.5.0.
It does not occur when downgrading to 1.4.0.

```
> nuxt-ts build


 FATAL  Cannot convert undefined or null to object                                                                                   13:14:00

  at Function.keys (<anonymous>)
  at getKeys (node_modules/deepmerge/dist/cjs.js:61:16)
  at mergeObject (node_modules/deepmerge/dist/cjs.js:86:2)
  at deepmerge (node_modules/deepmerge/dist/cjs.js:117:10)
  at ModuleContainer.module (node_modules/nuxt-logrocket/lib/module.js:5:19)
  at ModuleContainer.addModule (node_modules/@nuxt/core/dist/core.js:235:34)
  at node_modules/@nuxt/utils/dist/utils.js:1846:43
  at ModuleContainer.ready (node_modules/@nuxt/core/dist/core.js:55:5)
  at Nuxt._init (node_modules/@nuxt/core/dist/core.js:715:5)


   ╭───────────────────────────────────────────────────────────╮
   │                                                           │
   │   ✖ Nuxt Fatal Error                                      │
   │                                                           │
   │   TypeError: Cannot convert undefined or null to object   │
   │                                                           │
   ╰───────────────────────────────────────────────────────────╯
```

Do you know how to deal with it?

**Comment by farzadso** [comment]
@masayuki0220 Can you please provide a code sandbox or example?

**Comment by farzadso** [comment]
@masayuki0220 I think I found a way to recreate the problem. You're not providing the `logRocket` config in your `nuxt.config.js` file. I added a condition to check its presence.

This should be resolved in `v1.5.2`.

Please re-open if the issue exists.

**Comment by masayuki0220** [comment]
Thank you very much for your support!
